backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -429,6 +428,25 @@ public boolean addBreadcrumb(String message, Map<String, Object> attributes, Bac
         return database.getBreadcrumbs().addBreadcrumb(message, attributes, type, level);
     }
 
+    /**
+     * Enable client-side callstack resolution
+     * @param path              path to use for the socket file if DUMPWITHOUTCRASH unwinding is used, can be null
+     * @return true on success, otherwise returns false
+     */
+    public boolean enableClientSideUnwinding(String path) {
```

Comment:
Why do we need to pass a path to client side unwinding specific file. Why we can't maintain it inside backtrace-database directory or inside crashpad dir? In both examples - please remember we need to make sure directory exist. If we want to let users use it - we can also make sure they can't shoot into their foot via our API.

Reply:
You're right, I think we can always just use the same path for the bcd socket file as we will use for the crashpad database directory. Any objection to this @KrzaQ ?